t/plugin/jwt-auth.t

```diff
@@ -572,6 +572,8 @@ property "key" is required
     }
 --- response_body
 passed
+--- skip_eval
+1: $ENV{OPENSSL_FIPS} eq 'yes'
```

Comment:
Why add this?

Reply:
Openssl 3.0 with FIPS enabled does not support keys smaller than 2048bit.